messaging/WFLY-9213_Implement_the_Pause_method_for_a_Topic.adoc

```diff
@@ -0,0 +1,102 @@
+= Implement the Pause method for a Topic via Management APIs
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+
+An Artemis Queue can be paused but there is nothing for a Topic. The goal of this feature is to be able to pause a Topic by pausing all its consumers . New subscriptions being registered while the topic is in pause will be paused also.
```

Comment:
@ehsavoie What does it mean to "pause" a topic? Does it mean no messages will be received by subscribers? If it does what happen to these messages?

Reply:
Quoting Artemis documentation:
The AddressControl can pause and resume an address and all the queues that are bound to it. Newly added queue will be paused too until the address is resumed. Thus all messages sent to the address will be received but not delivered. When it is resumed, delivering will occur again.

Reply:
Thanks, I failed to find this in docs.